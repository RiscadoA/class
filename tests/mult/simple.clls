proc main() {
	cut {
		println("send");
		send x(y. println("close y"); close y);
		println("wait x");
		wait x;
		println("end");
		()
		|x: recv wait; close|
		println("recv");
		recv x(y);
		println("wait y");
		wait y;
		println("close x");
		close x
	}
};;
