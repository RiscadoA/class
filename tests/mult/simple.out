send
close y
recv
wait y
close x
wait x
end
